README.rst

```diff
@@ -27,6 +27,7 @@ Tahoe is all about subdomains, so please add the following entries to your ``/et
     127.0.0.1 red.localhost
     127.0.0.1 blue.localhost
     127.0.0.1 green.localhost
+    127.0.0.1 edx.devstack.lms
```

Comment:
We need this as AMC requests `edx.devstack.lms:18000/appsembler/api/username/<username>/` while setting up the user.